src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const arrRows = table.rows;
+
+[...arrRows].forEach((row, i) => {
```

Comment:
Don't need to use index
```suggestion
[...arrRows].forEach((row) => {
```